sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -53,6 +53,8 @@
 public abstract class WritableColumnVector extends ColumnVector {
   private final byte[] byte8 = new byte[8];
 
+  protected abstract void releaseMemory();
```

Comment:
@cloud-fan do we treat `WritableColumnVector` as public api? if so, we should give it a default implementation instead of abstract method, otherwise, third-party subclass without implementing this method will be failed with

```
java.lang.AbstractMethodError: org.apache.spark.sql.execution.vectorized.WritableColumnVector.releaseMemory()V
	at org.apache.spark.sql.execution.vectorized.WritableColumnVector.close(WritableColumnVector.java:92)
	at io.glutenproject.vectorized.ArrowWritableColumnVector.close(ArrowWritableColumnVector.java:362)
```

Reply:
It's not a public API, I think third-party lib should update and re-compile the code when upgrading Spark versions if private APIs were used.

Reply:
got it, thanks for the information.